src/Agent.Sdk/Util/RepositoryUtil.cs

```diff
@@ -93,7 +135,7 @@ public static string GuessRepositoryType(string repositoryUrl)
 
                 return RepositoryTypes.Tfvc;
             }
-            else if (repositoryUrl.IndexOf("bitbucket.org", StringComparison.OrdinalIgnoreCase) >= 0)
+            else if (repositoryUrl?.IndexOf("bitbucket.org", StringComparison.OrdinalIgnoreCase) >= 0)
```

Comment:
I missed this the first time, but should we early out up top if repositoryUrl is null or empty string?

Reply:
that's a good point. I will fix it.